docs/UserGuide.md

```diff
@@ -333,6 +358,18 @@ Format: `a`
 Hides the answer of the current flashcard. <br>
 Format: `h`
 
+### Review flashcard as correct : `t`
+
+Marks that the user got the answer correct for the current flashcard. <br>
+Note: This command can only be executed if the answer of the current flash card is shown. <br>
+Format: `t`
+
+### Review flashcard as wrong : `f`
```

Comment:
Remember to add these 2 commands in the table of content.

Reply:
Added. Also fixed the broken links in the table of content.